templates/docs/building-vanilla.md

```diff
@@ -118,10 +118,6 @@ module.exports = {
           {
             // Loads a SASS/SCSS file and compiles it to CSS
             loader: 'sass-loader',
-            options: {
```

Comment:
@andesol Are you sure this can be removed? I think `includePaths` was needed to make sure `.scss` files from within Vanilla files in node_modules are properly loaded as well.

Reply:
It's possible to pass options nesting them inside `sassOptions`, like this:
```            
loader: "sass-loader",
options: {
   sassOptions: {
      includePaths: ["absolute/path/a", "absolute/path/b"],
   }
```
But I found that to be unnecessary since webpack heads to `node_modules` when resolving imports, as stated in the [documentation](https://webpack.js.org/loaders/sass-loader/#resolving-import-at-rules): 

> The loader will first try to resolve @import as a relative path. If it cannot be resolved, then the loader will try to resolve @import inside node_modules.

Reply:
Great, thanks for detailed answer @andesol, when using Vanilla internally we rarely need to set up new webpack project with it, so our docs here got a bit rusty :)